client.go

```diff
@@ -119,7 +119,7 @@ type Marathon interface {
 	// remove a events listener
 	RemoveEventsListener(channel EventsChannel)
 	// remove our self from subscriptions
-	UnSubscribe() error
+	UnSubscribe(string) error
```

Comment:
Since we are changing the function anyway: would you like to rename it to `Unsubscribe`? :) http://www.thefreedictionary.com/unsubscribe
(sorry for this type of comment) :)